### ISSUE TITLE
Allow excluding a post from email, or exclude from the blog

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#77254]
When writing a blog post or newsletter issue, users want to be able to exclude the post from email delivery, or send the email but exclude the post from the public blog.

This comes up semi-frequently, some examples:
- p1684787107103569-slack-C052XEUUBL4
- p58i-euJ-p2#comment-58221
- p1HpG7-ncr-p2
- p1686882865579699-slack-C052XEUUBL4
- p7pQDF-8Ms#comment-23733
- p1691057803217889-slack-C052XEUUBL4

**Comment by katiebethbrown** [comment]
Related:
https://github.com/Automattic/jetpack/issues/10876

**Comment by simison** [comment]
Excluding from email used to be possible with a filter in old post editor: https://jetpack.com/blog/hook-month-control-subscriptions/

Those filters might still programmatically work.

**Comment by crisbusquets** [comment]
@simison I worked with Francisco and Michael to evolve the current Gutenberg newsletter panel.

I opted to remove `Everyone`, since it isn't easy to understand the difference with `Anyone subscribed` (especially if we _merge_ followers/subscribers).

This is the version (current vs proposal). Figma file (h8tMpJlCqNFemEerU9owHI-fi-3266_125166).

<img width="630" alt="Screenshot 2023-07-07 at 15 30 17" src="https://github.com/Automattic/wp-calypso/assets/25607244/9b8916eb-e964-4f50-9048-f0dd45922f1f">


After reading again this Slack thread (p1687789942117879/1687787662.747329-slack-C052XEUUBL4), seems that excluding it from the blog will require changes to Core.
